.PVS-Studio/project.json

```diff
@@ -0,0 +1,530 @@
+{
+  "modules": [
+    {
+      "name": "proj-happycows-f25-15",
+      "path": "/Users/guntashgill07/cs156/proj-happycows-f25-15/",
+      "sources": [
+        "/Users/guntashgill07/cs156/proj-happycows-f25-15/src/main/java/edu/ucsb/cs156/happiercows/HappierCowsApplication.java",
+        "/Users/guntashgill07/cs156/proj-happycows-f25-15/src/main/java/edu/ucsb/cs156/happiercows/advice/HappierCowsControllerAdvice.java",
```

Comment:
This file, given that it has hard coded paths of /Users/guntash, etc. should probably be added to the .gitignore, and should definitely not be merged into the repo (at least not in this form).